doc/source/release-notes/Percona-Server-5.7.18-16.rst

```diff
@@ -0,0 +1,103 @@
+.. rn:: 5.7.18-16
+
+==========================
+|Percona Server| 5.7.18-16
+==========================
+
+Percona is glad to announce the GA (Generally Available) release of |Percona
+Server| 5.7.18-16 on July 28, 2017 (Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.18-16/>`_
+and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.7.18
+<http://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-18.html>`_, including
+all the bug fixes in it, |Percona Server| 5.7.18-16 is the current GA release
+in the |Percona Server| 5.7 series. All of |Percona|'s software is open-source
+and free, all the details of the release can be found in the `5.7.18-16
+milestone at
+Launchpad <https://launchpad.net/percona-server/+milestone/5.7.18-16>`_
+
+New Feature
+===========
+
+.. Do we mention that x64 is the only supported platform.
+
+|Percona Server| is now available on Debian 9 (stretch). The support only covers
+the ``amd64`` architecture.
+
+|Percona Server| can now be built with support of SSL 1.1.
+
+|TokuDB| enables to kill a query that is awaiting an FT locktree lock.
+
+|TokuDB| eables using the ``MySQL DEBUG_SYNC`` facility within |Percona FT|.
+
+Bugs Fixed
+==========
+
+Row counts in |TokuDB| could be lost intermittently after restarts. Bug fixed
+:tdbbug:`2`.
+
+In |TokuDB|, two races in the fractal tree lock manager could significantly
+affect transactional throughput for some applications that used a small number
+of concurrent transactions.  These races manifested as transactions
+unnecessarily waiting for an available lock. Bug fixed :tdbbug:`3`.
+
+|Percona FT| could assert when opening a dictionary with no useful information
+to error log. Bug fixed :tdbbug:`23`.
+
+|Percona FT| could assert for various reasons deserializing nodes with no useful
+error output. Bug fixed :tdbbug:`24`.
+
+It was not possible to build |Percona Server| on Debian 9 (stretch) due to
+issues with openssl1.1. Bug fixed :bug:`1702903`.
```

Comment:
Taking into account that there is no such package in Debian (no full match at least) and also based on terminology usage in the upstream bug report I assume it is about OpenSSL 1.1